nav2_constrained_smoother/CMakeLists.txt

```diff
@@ -74,8 +81,7 @@ if(BUILD_TESTING)
   ament_lint_auto_find_test_dependencies()
 endif()
 
-ament_export_include_directories(include)
-ament_export_libraries(${library_name})
+ament_export_libraries(nav2_constrained_smoother)
```

Comment:
I think I should just add the export set and use `ament_export_targets`, but it's not strictly need to get this working on lunar.